requirements.txt

```diff
@@ -1,17 +1,17 @@
-hiredis==0.1.1
-redis==2.9.0
+hiredis==0.1.5
+redis==2.10.3
 mock==1.0.1
-Werkzeug==0.9.1
-PyYAML==3.10
-Flask==0.10
-Flask-SeaSurf==0.1.13
-Flask-Assets==0.8
-fakeredis==0.3.0
-decorator==3.3.2
-jsmin==2.0.2-1
-python-dateutil==1.5
-yuicompressor==2.4.7
-closure==20121212
-Flask-DebugToolbar==0.7.1
-Markdown==2.3.1
+Werkzeug==0.10
```

Comment:
Does this change actually work? We've had issues with applications breaking on upgrade to Werkzeug 0.10 from 0.9.1 under both gunicorn and the normal flask wsgi server.